### ISSUE TITLE
aria-label should be an expression

### ISSUE THREAD

**Issue by kschwidder** [angular/material#1639]
aria-label is currently a string which does not play nicely with multi-lingual apps. It should be an expression or string. In common apps you will set the label as {{LABEL | translate}}

**Comment by marcysutton** [comment]
Hi, which component(s) are you talking about? Not everything uses aria-label.

**Comment by kschwidder** [comment]
md-button primarily

**Comment by marcysutton** [comment]
@kschwidder I was able to use angular-translate successfully for the `aria-label` on md-button with no problems–the text swapped out when I changed the preferred language. Can you provide more details on your situation? Are you actually finding errors or restrictions?

**Comment by kschwidder** [comment]
Can you share the code example ? I have tried it with aria-label="{{'MYLABEL' | translate}}".

**Comment by marcysutton** [comment]
I followed the docs on http://angular-translate.github.io/docs/#/guide and made local changes to our button demos to prove it worked (without committing). It looked something like this:

```js
var app = angular.module('myApp', ['ngMaterial', 'pascalprecht.translate']);
app.config(['$translateProvider', function ($translateProvider) {
  $translateProvider.translations('en', {
    'MYLABEL': 'Hello'
  });
  $translateProvider.translations('de', {
    'MYLABEL': 'Hallo'
  });
  $translateProvider.preferredLanguage('en');
}]);
```
```html
<md-button aria-label="{{MYLABEL | translate}}"><span class="icon"></span></md-button>
```

**Comment by kschwidder** [comment]
Will check again. Thx ! Will close the issues

**Comment by mxab** [comment]
Hi, would it be better to have a ```aria-label-translate="CAKE"``` directive because as far as I understand it the ```translate``` filter is stateful and therefore should be not used to often?

**Comment by johnrix** [comment]
[Here's](https://www.google.de/url?sa=t&rct=j&q=&esrc=s&source=web&cd=4&cad=rja&uact=8&ved=0ahUKEwjtj7_Y3MXMAhXFBywKHbfjBlEQFggxMAM&url=https%3A%2F%2Fjsfiddle.net%2Fsscovil%2Fe0dsdut7%2F&usg=AFQjCNEDVMwOmAbCHkFTlyLK6dkpHl7nvA&sig2=HLpsfuHNLnu2jbbzfZ_Baw) a directive for that which I stumbled on.
